features/vesting/hooks.ts

```diff
@@ -173,3 +243,45 @@ export const useVestingSnapshotDelegate = () => {
     [contractWeb3, chainId],
   );
 };
+
+export const useRevokeUnvested = (escrow: string | undefined) => {
+  const { chainId } = useWeb3();
+  const { contractWeb3 } = useVestingEscrowContract(escrow);
+
+  return useCallback(async () => {
+    if (chainId == null || contractWeb3 == null) {
+      return;
+    }
+    await transaction('Revoke unvested tokens', chainId, () =>
```

Comment:
you sure you do not need return value here?

Reply:
It's not used anywhere 🤔